package-lock.json

```diff
@@ -28,9 +28,9 @@
       }
     },
     "ajv": {
-      "version": "5.3.0",
```

Comment:
I'm not a big fan of committing this kind of file, but I don't know how `package-lock.json` file works exactly, so I'm ok with this.

These changes could be done in a different PR, but I don't want to delay more the PR because of this personal preference.

So, ok, no change required.

Reply:
I committed this after my Travis build failed with dependency problems.  I'm not sure if it's necessary or not-- it might be that my `node_modules` contained something out of date.  

I noticed that the Node version is Travis seemed to be unpinned and my CI build ran under NodeJS 9.2.  I'm wondering if we've made a mistake in not being specific about that version to prevent mysterious issues with the builds.   What do you think?

Reply:
I thought someone on @exercism/ecmascripts configured Node version to use the latest LTS (I'm not blaming, so I don't want to look for who was exactly). So, Node.js v8 should be used in Travis CI.

We can have `package-lock.json` in this PR, don't waste time with this.

Reply:
Thanks for having a look Rubén!